### PR TITLE
Release note for Feb 2023 new global config for run step commands

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Creating a release note for new global config for partial parsing which has been added to dbt Cloud. 

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Allow for `--no-partial-parse` global config in run step commands"
+id: "no-partial-parse-config"
+description: "New run step command"
+sidebar_label: "New: Run step global config"
+tags: [Feb-15-2023]
+---
+
+tbd
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md]:
```suggestion
You can now enable partial parsing in dbt Cloud run step commands. Previously, the [`--no-partial-parse` global config](/reference/global-configs#partial-parsing) was only available in dbt Core. For more information on partial parsing, refer to [partial parsing](/reference/parsing#partial-parsing).
```

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Allow for `--no-partial-parse` global config in run step commands"
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md]:
```suggestion
title: "Partial parsing available in dbt Cloud run step commands"
```

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+id: "no-partial-parse-config"
+description: "New run step command"
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md]:
```suggestion
description: "You can now enable partial parsing in dbt Cloud run steps."
```

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+sidebar_label: "New: Run step global config"
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md]:
```suggestion
sidebar_label: "New: Partial parsing in run steps"
```

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Partial parsing available in dbt Cloud run step commands"
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md]:
```suggestion
title: "Partial parsing available in dbt Cloud run steps"
```

**Review by schottj — DISMISSED** [review]
thank you!

File: website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Partial parsing available in dbt Cloud run steps"
+id: "no-partial-parse-config"
+description: "You can now enable partial parsing in dbt Cloud run steps."
+sidebar_label: "New: Partial parsing in run steps"
+tags: [Feb-15-2023]
+---
+
+You can now enable partial parsing in dbt Cloud run step commands. Previously, the [`--no-partial-parse` global config](/reference/global-configs#partial-parsing) was only available in dbt Core. For more information on partial parsing, refer to [partial parsing](/reference/parsing#partial-parsing).
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/11-Feb-2023/no-partial-parse-config.md]:
```suggestion
You can now enable partial parsing in dbt Cloud run step commands. Previously, the [`--no-partial-parse` global config](/reference/global-configs#partial-parsing) was only available in dbt Core. For more information, refer to [partial parsing](/reference/parsing#partial-parsing).
```

**Review by nghi-ly — COMMENTED** [review]
looks great! one minor nit

**Review by nghi-ly — APPROVED** [review]
:shipit:

**Pull request merged**